jwst/outlier_detection/outlier_detection_step.py

```diff
@@ -102,7 +102,8 @@ def process(self, input_data):
 
             # Setup outlier detection parameters
             pars = {
-                'weight_type': self.weight_type,
+                'weight_type': self.weight_type,  # for calling the resample step
+                'wht_type': self.weight_type,  # for calling the resample class directly
```

Comment:
@melanieclarke This list is strictly internal use only, right?  i.e. we're not going to be presenting users with a confusing set of names of both `weight_type` and `wht_type`

Reply:
Yes, this is only used internally.  The top-level parameter is "weight_type" only.